### PR TITLE
Version 0.29 + noarch

### PR DESCRIPTION
**Pull request by rth**
* Update to the latest version v0.29
 * [switch to `noarch`](https://conda-forge.org/docs/meta.html#building-noarch-packages)
 * add myself as a maintainer

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, please rerender

**Comment by rth** [comment]
Thanks @isuruf , was going to do that as a magical solution to everything )

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% set name = "apispec" %}
-{% set version = "0.27.1" %}
-{% set sha256 = "878e525c6f171afdb5ac99001c05a4fdc2d2ede373fe6c862f282940b2f9ab19" %}
+{% set version = "0.29" %}
```
Comment (isuruf) [inline recipe/meta.yaml]: `0.29.0`

Comment (rth) [reply recipe/meta.yaml]: Thanks for the review. Should be fixed now.

**Comment by jakirkham** [comment]
Has some merge conflicts it appears. Any interest in this @frol?

**Comment by frol** [comment]
I will take care of it!

@rth I am sorry that I have never reviewed this! I don't know how this PR slipped out of my table.

**Comment by rth** [comment]
No worries @frol ..

Do we still want to merge this though? Given that the last version is v0.37 and this would be v0.29, the easiest I think is to close this and open a new one for v0.29 if anyone needs it..

**Comment by frol** [comment]
Yes, that is definitely the easiest way to go. It should be as easy as just throw `noarch: python`, I believe.

**Pull request closed without merging**